src/main/java/Branch.java

```diff
@@ -0,0 +1,48 @@
+public class Branch {
+
+    private int ID;
+    private String Name;
+    private String Address;
+    private String Owner;
+    private Exam exams;
+
+    public int getID() {
```

Comment:
Don't create getters and setters without reason.
create only what you need - if you create the object using json so im not sure you need these setters at all.

Reply:
removed setters. using the builder instead.